src/App.jsx

```diff
@@ -81,13 +82,21 @@ function App() {
     setVolume(0);
   }
 
+  function handleAnswer(e) { //set new song to user input, default to the current day if invalid input, or the previous answer if neither is valid
+    let newSongNumber = !testMode ? Math.min(e.target.value - 1, setGameIndex()) : e.target.value - 1;
```

Comment:
For some reason I can't toggle past song 211, although there are technically 216 songs.

Reply:
e.g. if I enter "215" it still plays song 211, although there are 216 songs in total.

Reply:
That's intended, to prevent skipping past the current day's puzzle. To get rid of that functionality you just get rid of the Math.min() on line 86.

Reply:
Oh nvm, good point!